Source/relay/TourGuide/Items of the Month/2019/Kramco Sausage-o-Matic.ash

```diff
@@ -41,17 +43,19 @@ void IOTMKramcoSausageOMaticGenerateResource(ChecklistEntry [int] resource_entri
     string main_title;
     
     KramcoSausageFightInformation fight_information = KramcoCalculateSausageFightInformation();
+    
+    int kramcosEquipped = lookupItem("Kramco Sausage-o-Matic&trade;").equipped_amount() + lookupItem("replica Kramco Sausage-o-Matic&trade;").equipped_amount();
```

Comment:
We end up with this pattern a few places where we have to lookup and check both pieces of equipment. IMO it'd be nice to have a function with signature `boolean itemOrReplicaEquipped(string itemName)` implementing this logic that we could reuse everywhere.

Reply:
you are probably right, but i'm not sure the benefit (which will improve... 5 uses of this sort of silly equip logic?) overrides the annoyance of doing it. i might do this, but it will depend how tired i am when i get to it lol

Reply:
i decided not to do this because it won't help us in the future -- i agree that this would be better i just am too tired and still have too much eagle correction to do lol